js/src/modal/modal.spec.js

```diff
@@ -3,7 +3,191 @@ import EventHandler from '../dom/event-handler'
 import { makeArray } from '../util/index'
 
 /** Test helpers */
-import { getFixture, clearFixture, createEvent, jQueryMock } from '../../tests/helpers/fixture'
+import { getFixture, clearFixture, unstyleFixture, createEvent, jQueryMock } from '../../tests/helpers/fixture'
+
+const getModalCss = () => {
+  return [
```

Comment:
it's a lot of CSS, is it really needed? If it's just to emulate the overflow for me it's a bit too much

Reply:
No it's not all needed, I have it narrowed down.

Reply:
Thanks that's better! 👍